### DISCUSSION TITLE
Qutebrowser 3 on ARM Linux

### DISCUSSION THREAD

**Discussion by stefanpinterBE** [qutebrowser/qutebrowser#8097] [Q&A] [answered]
hi!

i want to run qutebrowser 3 and higher in my Linux (Ubuntu 23.10 ARM) VM but the installation fails

i run this: python3 scripts/mkvenv.py

i get this: aarch64 is not a supported architecture for PyQt binaries on linux, this will most likely fail.

qutebrowser 2.5 is working (it is in the Ubuntu repository)

any suggestions? would it make sense to make it work with pyqt5?

**Comment by The-Compiler** [comment, marked as answer]
Please use discussions for things that aren't bugs or feature requests - I converted your issue into one.

You could use `--pyqt-type link`, which uses your system-wide Qt and PyQt. Looks like Ubuntu 23.10 ships with Qt 6.4 which isn't too shabby (September 2022).

**Reply by stefanpinterBE** [reply]
this actually worked, thank you! :)

```
sudo apt install python3-pyqt6 python3-pyqt6.qtwebengine
git clone https://github.com/qutebrowser/qutebrowser
cd qutebrowser
/usr/bin/python3 ./scripts/mkvenv.py --pyqt-type link
```

then run qutebrowser like this:

.venv/bin/qutebrowser
